comp_sci_gender_bias/utils/process_pandas.py

```diff
@@ -23,3 +23,59 @@ def remove_nonalphanum_lowercase(column: pd.Series) -> pd.Series:
         .str.replace("[^a-z0-9 ]", "", regex=True)
         .str.strip()
     )
+
+
+def remove_fw_slash(df: pd.DataFrame, col: str) -> pd.DataFrame:
+    """Remove '/' from the end of all rows in specified
+    df and col"""
+    df[col] = df[col].map(lambda x: str(x)[:-1] if str(x)[-1] == "/" else x)
+    return df
+
+
+def remove_http_https(df: pd.DataFrame, col: str) -> pd.DataFrame:
+    """Remove http:// and https:// from the start of all rows in
+    specified df and col"""
+    df[col] = df[col].map(lambda x: str(x).replace("https://", ""))
+    df[col] = df[col].map(lambda x: str(x).replace("http://", ""))
+    return df
+
+
+def remove_new_para(df: pd.DataFrame, col: str) -> pd.DataFrame:
+    "Remove \n from specified df and col"
+    df[col] = df[col].map(lambda x: str(x).replace("\n", ""))
+    return df
+
+
+def add_www(df: pd.DataFrame, col: str) -> pd.DataFrame:
+    """Add www. to start of all rows in specified
+    df and col"""
+    df[col] = df[col].map(lambda x: f"www.{str(x)}" if str(x)[:4] != "www." else x)
```

Comment:
is applying `str` needed? also see suggestion above using `.endswith`

Reply:
Yes it does need `str`. I've updated it to use `.startswith`